src/scribe.js

```diff
@@ -155,7 +155,7 @@ define([
     if (skipFormatters) {
       this._skipFormatters = true;
     }
-    this.el.innerHTML = html;
+    if (this.el.innerHTML !== html) this.el.innerHTML = html;
```

Comment:
Pretty minor but can we get some brackets here and can we get a comment explaining this is for IE only? Other than that looks good!